### ISSUE TITLE
Lighter weight eneric overpass downloader?

### ISSUE THREAD

**Issue by rafaqz** [DeloitteDigitalAPAC/LightOSM.jl#55]
Hi, this package looks great. But much more than what I need. It would be nice have a package that just wraps overpass queries so we can do them from julia.

Any chance the downloads could be separated out into a sub-package and genrealised to any object type? (e.g. I need rivers...). This could be in the same repo but with only the HTTP dependency, and registered separately?

**Comment by mmiller-max** [comment]
Hi, thanks for the idea, that would be a great addition to the Julia community! The splitting out would be relatively easy, but the generalisation would be a bit of work obviously. It could be done in isolation from `LightOSM` too and which could then transfer when ready. Not sure how much capacity @captchanjack has at the moment (I don't think very much) and I'm stretched as well, but hopefully one of us could do it at some point unless someone else gets there first.

**Comment by rafaqz** [comment]
Yes, it would great to have this, probably as part of JuliaGeo. I havent had time so far, but I can assist with reviews etc if anyone else does.
